dev/CommonStyles/TestUI/CompactPage.xaml.cs

```diff
@@ -75,7 +75,7 @@ private void RunTest_Click(object sender, RoutedEventArgs e)
             item = TreeView.FindVisualChildByType<TreeViewItem>();
             VerifyHeight(simpleVerify, item, 24, "TreeViewItem");
 
-            VerifyHeight(simpleVerify, NavItem1, 32, "NavigationViewItem");
+            VerifyHeight(simpleVerify, NavItem1, 36, "NavigationViewItem");
```

Comment:
in Compact.xaml it's still 32:
![image](https://user-images.githubusercontent.com/21356912/107583107-538b0280-6baf-11eb-974d-28505b5b2711.png)
Isn't it weird to set it to 32 there and compare to 36 here?

Reply:
it is set as 32ps in Compact.xaml, but evaluates to 36px in the test as it takes into account the 2px margins on both top and bottom.